### ISSUE TITLE
`get_global_id` does not work if function is directly exported

### ISSUE THREAD

**Issue by geexie** [IntelPython/numba-dpex#693]
If I try the following code

```
import dpctl
import base_l2_distance
import math

from numba_dppy import kernel, get_global_id, atomic, DEFAULT_LOCAL_SIZE
atomic_add = atomic.add

@kernel(access_types={"read_only": ["a", "b"], "write_only": ["c"]})
def l2_distance_kernel(a, b, c):
    i = get_global_id(0)
    j = get_global_id(1)
    sub = a[i, j] - b[i, j]
    sq = sub**2
    atomic_add(c, 0, sq)

def l2_distance(a, b, distance):
    with dpctl.device_context(base_l2_distance.get_device_selector()):
        l2_distance_kernel[(a.shape[0], a.shape[1]), DEFAULT_LOCAL_SIZE](a, b, distance)

    return math.sqrt(distance)

```
the following error is risen during typeinfer

```
  File "/nfs/site/home/mkolpako/code/miniconda3/envs/gpu_dev/lib/python3.7/site-packages/numba/core/typeinfer.py", line 1466, in resolve_value_type
    raise TypingError(msg, loc=inst.loc)
numba.core.errors.TypingError: Failed in dppy_nopython mode pipeline (step: nopython frontend)
Untyped global name 'get_global_id': Cannot determine Numba type of <class 'function'>

File "l2_distance_kernel.py", line 59:
def l2_distance_kernel(a, b, c):
    i = get_global_id(0)
```

Everything works if I use `numba_dppy.get_global_id` notation

**Comment by PokhodenkoSA** [comment]
Environment:
dpctl 0.11.1
numba 0.54.1
numba-dppy 0.15.0

**Comment by PokhodenkoSA** [comment]
Addressed in #694.
